translations/russian/research/notes.md

```diff
@@ -0,0 +1,16 @@
+## Using Russian in web apps
+
+* Most of web apps use more formal, neutral words.
+* Apps use infinitive for common verbs.
+
+*Log in - Войти.*
+*Sign up - Зарегестрироваться.*
+*Create - Создать.*
+
+* In web applications, it is better to refer to the second person plural
```

Comment:
I think this would be a little easier to understand:
```
In web applications, it is better to address the user with the second person
plural pronoun.
```
Don't forget the line break and the period at the end of the sentence :)